builds/misc/packages.yaml

```diff
@@ -0,0 +1,269 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+pr: none
+variables:
+  REVISION: '1'
+  VSTS_WORK: '/home/vsts/work'
+jobs:
+
+################################################################################
+  - job: ubuntu_amd64
+################################################################################
+    displayName: Ubuntu - amd64
+    pool:
+      name: Hosted Ubuntu 1604
+      vmImage: 'ubuntu-16.04'
+    steps:
+      - bash: |
+          echo "##vso[task.setvariable variable=RUSTUP_HOME;]$VSTS_WORK/rustup"
```

Comment:
Why do we care about this? They're hosted VMs anyway, so writing outside the working directory isn't a problem.

Reply:
This is from the time when VSTS jobs ran in containers. It was important to make sure that these directories are set properly. This may not be an issue anymore with the new queue.

Reply:
I've definitely been able to install them to the default `~/.cargo` in `ubuntu-16.04` VMs in my own projects.